src/pages/ctrader-signup/signup-success.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { form_style, paragraph_style } from './ctrader-form.module.scss'
+import SignupSuccessForm from './_signup-success-form'
+import { Localize, WithIntl } from 'components/localization'
+import EmailIcon from 'images/svg/check-email/email.svg'
+import { isBrowser } from 'common/utility'
+import CtraderWrapper from 'components/custom/_ctrader-wrapper'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import Button from 'features/components/atoms/button'
+import Link from 'features/components/atoms/link'
+import Layout from 'features/components/templates/layout'
+import { useResidenceList } from 'components/hooks/use-residence-list'
+
+const SignupSuccess = () => {
+    const [show_check_email, setShowCheckEmail] = useState(true)
+    const url_params = new URLSearchParams((isBrowser() && window.location.search) || '')
+    const email = url_params.get('email')
+    const [residence_list] = useResidenceList()
+
+    return (
+        <Layout>
+            <CtraderWrapper>
+                <Flex.Box
+                    as="div"
+                    justify="center"
+                    align="center"
+                    direction="col"
+                    gap="15x"
+                    className={form_style}
+                >
+                    <Typography.Heading as="h3" size="small" align="center" weight="bold">
```

Comment:
by default in heading
```suggestion
                    <Typography.Heading as="h3" size="small" align="center">
```